### PR TITLE
Provide import ID after a Md5sum collision 

### PR DESCRIPTION
**Pull request by Pheipp**
connected to https://github.com/Safecast/safecastapi/issues/486

The best way I came up with, is to change the message afterwards.

### CODE REVIEW

**Comment by Pheipp** [comment]
Well, sure. I will look what I can do.

**Comment by Pheipp** [comment]
Well, seems like I broke something and I have no idea what. Also that's the first time I wrote a spec so I am open for improvements.

**Comment by eitoball** [comment]
There was an issue when updating instances of `BgeigieImport`. I fixed the issue and moved spec as spec of `MeasurementImport`. Also,  I squashed commits and rebased on current master branch.

File: config/database.yml [hunk 1]
```diff
@@ -21,7 +21,9 @@ development:
   pool: 5
   username: safecast
   su_username: safecast
-  password: 
+  password:
+  host: postgresql
```
Comment (matschaffer) [inline config/database.yml]: We might want to be careful about adding this. Pretty sure cloud66 uses the checked in database.yml. Should probably back out the change until we verify or move the app to AWS

**Pull request merged**